kafka-connect-neo4j/src/test/kotlin/streams/kafka/connect/sink/Neo4jValueConverterTest.kt

```diff
@@ -174,11 +179,55 @@ class Neo4jValueConverterTest {
                 "string" to string,
                 "date" to date))
 
-        assertEquals(double, result["double"]?.asDouble())
-        assertEquals(long, result["long"]?.asLong())
-        assertEquals(bigDouble.toPlainString(), result["bigDouble"]?.asString())
-        assertEquals(string, result["string"]?.asString())
-        assertEquals(date.toInstant().atZone(ZoneId.of("UTC")).toLocalDateTime(), result["date"]?.asLocalDateTime())
+        assertEquals(double, result["double"])
+        assertEquals(long, result["long"])
+        assertEquals(bigDouble.toPlainString(), result["bigDouble"])
+        assertEquals(string, result["string"])
+        assertEquals(date.toInstant().atZone(ZoneId.of("UTC")).toLocalDateTime(), result["date"])
+    }
+
+    @Test
+    fun `should be able to process a CUD format AVRO structure`() {
```

Comment:
should we also include a test for relationship CUD format?

Reply:
done